packages/react-router-native/babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: ["module:metro-react-native-babel-preset"]
```

Comment:
I was running into some issues with React Native/Babel/Jest, and moving from a `.babelrc` file to `babel.config.js` helped for some reason :man_shrugging:.